src/components/Slider/SliderRange.tsx

```diff
@@ -329,7 +328,14 @@ const SliderRange: React.FC<SliderRangeProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```

Comment:
The `styles.track` is StyleSheet.absoluteFillObject so we do not need create this style

Reply:
Updated